### PR TITLE
Skip AssetPostProcessor when file in StreamingAssets folder. #166

### PR DESCRIPTION
**Pull request by ousttrue**

### CODE REVIEW

File: Assets/VRM/UniGLTF/Scripts/IO/UnityPath.cs [hunk 1]
```diff
@@ -44,6 +44,24 @@ public bool IsUnderAssetsFolder
...
+                {
+                    return false;
+                }
+
+                if (!FullPath.StartsWith(Application.streamingAssetsPath))
+                {
+                    return false;
+                }
+
+                return FullPath[Application.streamingAssetsPath.Length] == '/';
```
Comment (yutopp) [inline Assets/VRM/UniGLTF/Scripts/IO/UnityPath.cs]: This code may raise a `IndexOutOfRangeException`. For example, `UnityPath.FromUnityPath(Application.streamingAssetsPath).IsStreamingAsset` will be an edge case.

File: Assets/VRM/UniGLTF/Scripts/IO/UnityPath.cs [hunk 2]
```diff
@@ -44,6 +44,24 @@ public bool IsUnderAssetsFolder
...
+        public bool IsStreamingAsset
+        {
+            get
+            {
+                if (IsNull)
+                {
+                    return false;
+                }
+
+                if (!FullPath.StartsWith(Application.streamingAssetsPath))
```
Comment (yutopp) [inline Assets/VRM/UniGLTF/Scripts/IO/UnityPath.cs]:
```suggestion
                return FullPath.StartsWith(Application.streamingAssetsPath+"/");
```

File: Assets/VRM/UniGLTF/Scripts/IO/UnityPath.cs (hunk 1 above)
Comment (yutopp) [reply Assets/VRM/UniGLTF/Scripts/IO/UnityPath.cs]: How about a suggestion for a solution on line 56?

**Comment by yutopp** [comment]
This change resolves #166.

File: Assets/VRM/UniGLTF/Scripts/IO/UnityPath.cs (hunk 2 above)
Comment (ousttrue) [reply Assets/VRM/UniGLTF/Scripts/IO/UnityPath.cs]: 👍 
I avoid string `+`, but this one is safer

**Comment by ousttrue** [comment]
I rewrote.

**Pull request merged**